src/Gd/Imagine.php

```diff
@@ -91,7 +91,7 @@ public function open($path)
 
         $resource = $this->createImageFromString($data);
```

Comment:
The doc block of `createImageFromString` does not reflect the new type yet.

```
@return resource|\GdImage|false
```

Reply:
Fixed in e424a69a354dd46a57467a841128bad10bc249e2